scripts/git_branch.py

```
@@ -13,94 +13,88 @@
 
 
 def warn(msg):
-    print(f'WARNING [git_branch.py]: {msg}', file=sys.stderr)
+    print(f"WARNING [git_branch.py]: {msg}", file=sys.stderr)
 
 
 def run_git_value(project_dir, args, label):
     try:
         value = subprocess.check_output(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)